components/CheckInOutForm/index.tsx

```diff
@@ -0,0 +1,128 @@
+import {
+  Autocomplete,
+  Box,
+  FormControl,
+  IconButton,
+  TextField,
+  Typography,
+} from '@mui/material'
+import { DateTimePicker } from '@mui/x-date-pickers'
+import dayjs, { Dayjs } from 'dayjs'
+import React from 'react'
+import { ItemDefinition, User } from '../../utils/types'
+import AddOutlinedIcon from '@mui/icons-material/AddOutlined'
+import RemoveOutlinedIcon from '@mui/icons-material/RemoveOutlined'
+import colors from '../../utils/colors'
+
+interface Props {
+  kioskMode: boolean
+  users: User[]
+  itemDefinitions: ItemDefinition[]
+}
+
+function CheckInOutForm({ kioskMode, users, itemDefinitions }: Props) {
+  const [date, setDate] = React.useState<Dayjs | null>(dayjs(new Date()))
+  const [quantity, setQuantity] = React.useState<number>(1)
+  const [selectedStaff, setSelectedStaff] = React.useState<User | null>()
+  const [selectedItemDefinition, setSelectedItemDefinition] =
+    React.useState<ItemDefinition | null>()
+
+  return (
+    <>
+      <FormControl fullWidth>
+        {/* Staff member, Date, and Item input fields */}
+        {kioskMode && (
+          <Autocomplete
+            options={users.map((user) => user.name)}
```

Comment:
will this set the val as the name? do we maybe want make the val for this the id so its easier to pass to mongo?